SECURITY_FEATURES.md

```diff
@@ -41,13 +41,25 @@ Using partition sets and modeled settings removes the dependency on correct loca
 There is no package manager database or shared filesystem tree that can become corrupted and make the process non-deterministic.
 
 Our philosophy for variants is that the right time for an unexpected major version update to the kernel or orchestrator agent is "never".
+
+#### Kubernetes variants
+
 Each Kubernetes variant will be supported for approximately one year after it is first released.
 This follows the [Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) support policy.
 Newer LTS releases of the kernel may be introduced in newer variants, but not in older ones.
 
 We provide [a Kubernetes operator](https://github.com/bottlerocket-os/bottlerocket-update-operator) for automated updates to Bottlerocket.
 We recommend deploying it on your Kubernetes clusters.
 
+#### ECS variant
+
+We currently have a single variant for Amazon ECS.
+Newer LTS releases of the kernel may be introduced in newer variants, but not in the current one.
```

Comment:
Maybe we can future-proof this with:

```suggestion
Newer LTS releases of the kernel may be introduced in newer variants, but not in older ones.
```

That's already what we use in the kubernetes section and we will have more ecs variants

Reply:
I changed this from the Kubernetes text because we don't have more than one.  "Older ones" didn't make sense to me.